app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,38 @@
+class StatsController < ApplicationController
+  include ActionController::HttpAuthentication::Token::ControllerMethods
+
+  before_action :authenticate, :set_access_control_headers
+
+  def index
+    @audited_publications = PublicActivity::Activity.where(key: 'vacancy.publish')
+                                                    .group('trackable_id, key').count
+    @signin_audit = PublicActivity::Activity.where("key LIKE 'dfe-sign-in.%'").group('key').count
+
+    @published = Vacancy.published.count
+    @draft = Vacancy.draft.count
+    @trashed = Vacancy.trashed.count
+    @scraped = @published - (@audited_publications['vacancy.publish'] || 0)
+
+    @audit = Activity.order(created_at: :desc).page(page) if audit?
+  end
+
+  private
+
+  def page
+    params[:page] || 1
```

Comment:
Unclear why we need to paginate this endpoint when it just returns a summary of information.